vkconfig/resourcefiles/Validation - Shader Printf.json

```diff
@@ -46,11 +46,10 @@
                     "type": "multi_enum"
                 },
                 "enables": {
-                    "default": [
-                        "VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_EXT",
-                        "VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_RESERVE_BINDING_SLOT_EXT"
-                    ],
-                    "description": "Setting an option here will enable specialized areas of validation",
+                "default": [
+                     "VK_VALIDATION_FEATURE_ENABLE_DEBUG_PRINTF_EXT"
+                     ],
+                "description": "Setting an option here will enable specialized areas of validation",
```

Comment:
Is this description correct? Seems it shouldn't refer to validation at all, but just the debug-printf option.

Reply:
That description field is not for the current setting (debug printf), but the settings as a whole, of which there are 4 to choose from:
                     "VALIDATION_CHECK_ENABLE_VENDOR_SPECIFIC_ARM": "ARM specific validation",
                        "VK_VALIDATION_FEATURE_ENABLE_BEST_PRACTICES_EXT": "Best Practices warning checks",
                        "VK_VALIDATION_FEATURE_ENABLE_DEBUG_PRINTF_EXT": "Debug printf",
                        "VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_EXT": "GPU-Assisted Validation",
                        "VK_VALIDATION_FEATURE_ENABLE_GPU_ASSISTED_RESERVE_BINDING_SLOT_EXT": "Reserve a descriptorSet binding slot for internal use"

Reply:
Ah I was mistaken then.